core/dbt/tests/util.py

```diff
@@ -215,6 +215,11 @@ class TestProcessingException(Exception):
 def run_sql_with_adapter(adapter, sql, fetch=None):
     if sql.strip() == "":
         return
+
+    # make data type change if warehouse type is snowflake
+    if adapter.acquire_connection().type == "snowflake":
+        sql = sql.replace("BIGSERIAL", "BIGINT AUTOINCREMENT")
+
```

Comment:
Should this.
1. be its own small little pr? needed to make at least a few tests pass in snowflake in the new method
2. does this seem like the best method for grabbing the type in new testing format? couldn't iterate or call on a bunch of other methods or the object itself so want to make sure I didn't miss something

Reply:
We'd like to avoid having adapter-specific code in the testing utilities, so this is probably not the way to to go. Do you really need to specify the type of that column? Some of the older tests using a seed.sql file, when we could actually switch to a csv file, and which will allow us to not specify the type of the 'id' column in the test.

If that doesn't work, then we can create a Snowflake override of the models fixture in the Snowflake tests.